tilings/tiling.py

```diff
@@ -928,8 +811,11 @@ def sub_tiling(self, cells: Iterable[Cell], factors: bool = False) -> "Tiling":
             if (factors and ass.gps[0].pos[0] in cells)
             or all(c in cells for c in chain.from_iterable(gp.pos for gp in ass.gps))
         )
-        return self.__class__(obstructions, requirements, assumptions)
+        return self.__class__(
+            obstructions, requirements, assumptions, simplify=False, sorted_input=True
+        )
 
+    @cssmethodtimer("Tiling.find_factors")
```

Comment:
Why do we time the `find_factors` methods?

Reply:
In case it's slow? We can probably remove this.

Reply:
My question came from the fact that it's the only method not related to the initializer that we are timing. We might have added the timing by accident.

Reply:
I think it was used at one point, but maybe not anymore. We should remove it.

Reply:
Fixed. Added timing to Tiling._simplify_griddedperms.